types/deflate-js/test/deflate-js-tests.global.ts

```diff
@@ -0,0 +1,3 @@
+deflatejs.deflate(new Uint8Array([1, 2, 3, 4]), 3); // $ExpectType Uint8Array
```

Comment:
I might be missing something obvious, but is the underlying library consumable as a global?

https://unpkg.com/browse/deflate-js@0.2.3/index.js